vite.config.ts

```diff
@@ -1,16 +1,19 @@
 /// <reference types="vitest" />
 
-import path from 'path';
+import * as dotenv from 'dotenv';
 import { defineConfig } from 'vite';
 import tsconfigPaths from 'vite-tsconfig-paths';
 
+dotenv.config();
+
+const pkg = process.env.npm_package_name;
+console.log('vitest:', pkg);
```

Comment:
we should remove the console log

Reply:
I would keep it as a hint for later refactoring, let's discuss tomorrow